### PR TITLE
ticdc: update changefeed scheduler configs

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update changefeed scheduler configs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@overvenus Please invite a tech review. Thanks

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -91,11 +91,12 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
...
 # 将表按 Region 个数划分成多个同步范围，这些范围可由多个 TiCDC 节点同步。
-# 注意：
-# 1. 横向扩展功能目前为实验特性，不建议在生产环境中使用。
-# 2. 该参数只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
-# 3. TiCDC 不会将小于该参数 Region 个数的表划分成多个同步范围。
-# region-per-span = 50000
+# 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
+#
+# 设置为 "true" 以打开该功能。
+enable-table-across-nodes = false
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:97]: 这里可以改成  true 吗？和上面的注释保持一致。

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -244,12 +244,12 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 
 > **警告：**
 >
-> - 横向扩展功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
-> - TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
+> - TiCDC v6.7.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:247]: v6.7.0 应该是 v7.0 吧
```suggestion
> - TiCDC v7.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
```

File: ticdc/ticdc-changefeed-config.md (hunk 1 above)
Comment (overvenus) [reply ticdc/ticdc-changefeed-config.md:97]: 这个配置默认为 false

Comment (hfxsd) [reply ticdc/ticdc-changefeed-config.md:97]: OK，那加上这默认值的描述吧。

File: ticdc/ticdc-changefeed-config.md [hunk 3]
```diff
@@ -91,11 +91,12 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
...
 [scheduler]
 # 将表按 Region 个数划分成多个同步范围，这些范围可由多个 TiCDC 节点同步。
-# 注意：
-# 1. 横向扩展功能目前为实验特性，不建议在生产环境中使用。
-# 2. 该参数只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
-# 3. TiCDC 不会将小于该参数 Region 个数的表划分成多个同步范围。
-# region-per-span = 50000
+# 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
+#
+# 设置为 "true" 以打开该功能。
```
Comment (hfxsd) [inline ticdc/ticdc-changefeed-config.md:96]:
```suggestion
# 默认为 "false"。设置为 "true" 以打开该功能。
```

**Review by hfxsd — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -244,12 +244,12 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 
 > **警告：**
 >
-> - 横向扩展功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
-> - TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
+> - TiCDC v7.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:247]:
```suggestion
> TiCDC v7.0.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
```

File: ticdc/ticdc-changefeed-config.md [hunk 5]
```diff
@@ -91,11 +91,12 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
...
-# 注意：
-# 1. 横向扩展功能目前为实验特性，不建议在生产环境中使用。
-# 2. 该参数只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
-# 3. TiCDC 不会将小于该参数 Region 个数的表划分成多个同步范围。
-# region-per-span = 50000
+# 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
+#
+# 默认为 "false"。设置为 "true" 以打开该功能。
+enable-table-across-nodes = false
+# 在一张表上启用该功能，该表最少需要包含的 Region 个数。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:98]:
```suggestion
# 启用该功能后，该功能只对 Region 个数大于等于 `region-threshold` 值的表生效。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 6]
```diff
@@ -244,12 +244,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 
 > **警告：**
 >
-> - 横向扩展功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
-> - TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
+> - TiCDC v7.0.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:247]:
```suggestion
> TiCDC v7.0.0 仅支持在 Kafka 同步任务上开启大单表的横向扩展功能。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 7]
```diff
@@ -244,12 +244,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+region-threshold = 100000
 ```
+
+> **注意：**
+>
+> 一个表包含的 Region 个数可用如下 SQL 查询：
+>
+> ```sql
+> SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="库名" AND TABLE_NAME="表名" AND IS_INDEX=0;
+> ```
```
Comment (hfxsd) [inline ticdc/ticdc-sink-to-kafka.md:265]:
```suggestion
一个表包含的 Region 个数可用如下 SQL 查询：

```sql
SELECT COUNT(*) FROM INFORMATION_SCHEMA.TIKV_REGION_STATUS WHERE DB_NAME="database1" AND TABLE_NAME="table1" AND IS_INDEX=0;
```

```

File: ticdc/ticdc-changefeed-config.md [hunk 8]
```diff
@@ -91,11 +91,12 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
 
 [scheduler]
 # 将表按 Region 个数划分成多个同步范围，这些范围可由多个 TiCDC 节点同步。
-# 注意：
-# 1. 横向扩展功能目前为实验特性，不建议在生产环境中使用。
-# 2. 该参数只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
-# 3. TiCDC 不会将小于该参数 Region 个数的表划分成多个同步范围。
-# region-per-span = 50000
+# 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
+#
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:95]:
```suggestion
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by Frank945946 — APPROVED** [review]

**Pull request merged**